src/apps/components/juttle-editor.js

```diff
@@ -0,0 +1,33 @@
+import React from 'react';
+
+import AceEditor from 'juttle-react-ace';
+
+import 'brace/theme/monokai';
+import './juttle-mode/mode';
+
+class JuttleEditor extends React.Component {
+    render() {
+        var program = this.props.bundle ? this.props.bundle.program : '';
+        var RUN_JUTTLE = {
+            name: 'RUN_JUTTLE',
+            bindKey: {win: 'Ctrl-Enter',  mac: 'Command-Enter'},
+            exec: this.props.runJuttle
+        };
+        var commands = [RUN_JUTTLE];
+        return (
+            <AceEditor
```

Comment:
Can we add a scroll margin to the top, so the top line of text isn't so close the border. Using react-ace you would have to add a `ref="aceEditor"` tag to the `<AceEditor />` component, then in `componentDidMount` do something like this:
```javascript
this.refs.aceEditor.renderer.setScrollMargin(15, 0)
```